HW#9/Candles.py

```diff
@@ -0,0 +1,25 @@
+def solution(candle_number: int, make_new: int, all_candle=0,
+             ostatok=0) -> int:
```

Comment:
what if someone accidentally calls this function with ALL set of parameters?
e.g.:
`solution(5, 2, 2, 5)`

Do not give a user the possibility to user your function incorrectly